docs/_config.yml

```
@@ -42,15 +42,15 @@ remote_theme: rundocs/jekyll-rtd-theme@v2.0.10
 #   - vendor/ruby/
 
 # Document versioning
-version: master
+version: v0.4
 display_version_list: true
 
 # Release is the full released version number. Used to make external links to
 # point to the correct blobs in the Github repo. This is also the version shown
 # in the sidebar (top left corner of the page)
-release: master
+release: v0.4
 
 # Container image which to point to in the documentation
 container_image: gcr.io/k8s-staging-nfd/node-feature-discovery-operator:master
 # Operand Node Feature Discovery documentation version
-operand_version: master
+operand_version: v0.10.0
```

nit: We already have v0.10.1 so
```suggestion
operand_version: v0.10.1
```